server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -11,6 +11,11 @@ CobolVisitor.misspelledWord=A misspelled word, maybe you want to put %s
 CobolVisitor.paragraphNotDefined=The following paragraph is not defined: %s
 CobolVisitor.progIDIssueMsg=There is an issue with PROGRAM-ID paragraph
 CobolVisitor.subroutineNotFound=%s: Subroutine not found
+CobolVisitor.theSameGroup=Redefined element should belong to the same group: %s
+CobolVisitor.redefinedContainValue=Redefined clause cannot contain VALUE: %s
+CobolVisitor.redefineRedefined=Redefined variable cannot be redefined: %s
+CobolVisitor.redefineImmediatelyFollow=Redefined variable should immediately follow: %s
+CobolVisitor.levelsMustMatch=Redefined variable level must be the same: %s
```

Comment:
The according checks do not belong to the CobolVisitor, so they should be placed under "semantics"